src/main/java/org/jabref/logic/ai/chatting/model/JvmOpenAiChatLanguageModel.java

```diff
@@ -31,13 +31,11 @@ public class JvmOpenAiChatLanguageModel implements ChatLanguageModel {
 
     public JvmOpenAiChatLanguageModel(AiPreferences aiPreferences, HttpClient httpClient) {
         this.aiPreferences = aiPreferences;
-
```

Comment:
If there is no new functionality, then it would be better to leave this file as is